### PR TITLE
completed 1 step

### PR DESCRIPTION
**Pull request by sau9458**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+class Todo extends React.Component{
+  constructor(){
+    super();
+    this.state={
+      data:input.data
+    }
+  }
+render(){
+  const da=this.state.data.map((data,id)=><li className="list-group-item" key={data.id}><input type="checkbox"/>
```
Comment (arun-kumar-codes) [inline src/index.js:15]: user proper variable name

File: src/todo.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```
Comment (arun-kumar-codes) [inline src/todo.js:1]: is this file required?? if not delete this

Comment (sau9458) [reply src/todo.js:1]: yes sir this file is required

File: src/index.js (hunk 1 above)
Comment (sau9458) [reply src/index.js:15]: used proper variable name
